xml/release-notes.xml

```diff
@@ -311,6 +311,27 @@
 
   </sect2> -->
 
+  <sect2 xml:id="sec.upgrade.kernel">
+   <title>Unification of &sles; and &opensuseleap; kernel packaging</title>
+   <para>
+    The original kernel was split into three subpackages:
```

Comment:
* "was split" sounds like past tense, describing a state that no longer exists--"has been split" is (imo?) clearer
* "original kernel" sounds a bit off, especially since our kernel is just one variant of the Linux kernel, so I'd suggest "default kernel", since that is part of the package name anyway.
* "on openSUSE Leap" adds context 

```suggestion
    On &opensuseleap;, the default kernel has been split into three subpackages:
```